pkg/reconciler/events/pubsub/pubsub.go

```diff
@@ -169,10 +176,10 @@ func (r *Reconciler) updateStatus(ctx context.Context, desired *v1alpha1.PubSub)
 
 	if err == nil && becomesReady {
 		duration := time.Since(src.ObjectMeta.CreationTimestamp.Time)
-		r.Logger.Infof("PubSub %q became ready after %v", source.Name, duration)
+		logging.FromContext(ctx).Desugar().Info("PubSub became ready", zap.Any("after", duration))
 
 		if err := r.StatsReporter.ReportReady("PubSub", source.Namespace, source.Name, duration); err != nil {
-			logging.FromContext(ctx).Infof("failed to record ready for Storage, %v", err)
+			logging.FromContext(ctx).Desugar().Info("Failed to record ready for PubSub", zap.Error(err))
```

Comment:
Is this always unexpected? Seems like it should be Error level.

Reply:
Done! and for the other reconcilers